tests/jballerina-unit-test/src/test/resources/test-src/types/xml/xml-attribute-access-syntax.bal

```diff
@@ -24,3 +24,23 @@ function usePredefinedXMLNamespace() returns string|error? {
     xml x = xml `<root xmlns="the-url" attr="the attr" xml:space="preserve"></root>`;
     return x.'xml:space;
 }
+
+function testXMLAfterRemoveAttribute() {
+    map<string> attributes = {"{http://www.w3.org/2000/xmlns/}ns0":"http://sample.com/test","status":"online"};
+    xml:Element element = xml:createElement("{http://sample.com/test}bookStore", attributes, xml ``);
+    assert(element.toString(), "<ns0:bookStore xmlns:ns0=\"http://sample.com/test\" status=\"online\"></ns0:bookStore>");
```

Comment:
@rdhananjaya there is some deviation compared to io:println().
```
io:println(element) -> <ns0:bookStore xmlns:ns0="http://sample.com/test" status="online"/>
```
any specific reason for this?

Reply:
This is because the the stax implementation we have with jBallerina-uniut-test module is different from the one we have when we build a distribution pack.
Both of these are valid, just different serializations.

Reply:
Ah okay. Thanks.